chapter02/dtj/ex4

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/perl
+
+use warnings;
+
+print "Enter the first number: ";
+chomp($num1 = <STDIN>);
+print "Enter the second number: ";
+chomp($num2 = <STDIN>);
+$result = $num1 * $num2;
+print "$num1 x $num2 =  $result \n";
```

Comment:
-  Probably worth adding some validation checks for non-numeric data, the script fails when non-numerics are entered.